core/src/main/java/io/questdb/cairo/ColumnPurgeOperator.java

```diff
@@ -326,7 +330,7 @@ private boolean purge0(ColumnPurgeTask task, final ScoreboardUseMode scoreboardM
                 }
 
                 // Check if it's symbol, try remove .k and .v files in the partition
-                if (ColumnType.isSymbol(columnType)) {
+                if (ColumnType.isSymbol(columnType) || columnTypeRogue) {
                     if (isSymbolRootFiles) {
```

Comment:
It looks like this branch won't be taken when `columnTypeRogue` is true. Shouldn't we check the root partition and try to purge dangling symbol files in `VacuumColumnVersions`?